macosx/HBFilters+UIAdditions.m

```diff
@@ -312,6 +391,21 @@ - (NSArray *)denoiseTunes
     return filterParamsToNamesArray(hb_filter_param_get_tunes, HB_FILTER_NLMEANS);
 }
 
+- (NSArray *)sharpenTypes
+{
+    return @[@"Off", @"Unsharp", @"Lapsharp"];
+}
+
+- (NSArray *)sharpenPresets
+{
+    return filterParamsToNamesArray(hb_filter_param_get_presets, HB_FILTER_UNSHARP);
```

Comment:
Preset names are the same between the two sharpen filters, but the underlying settings are different.